_episodes/03-func.md

```diff
@@ -95,19 +95,19 @@ sum:=0;; for g in S do sum := sum + Order(g); od; sum/Size(S);
 
 > ## This works only for rapid prototyping
 >
-> * incidentally, one could copy and paste only a part of the code, and
+> * one could copy and paste only a part of the code, and
```

Comment:
In the new edition, this sounds like a limitation. The old text emphasised that this could happen by mistake.

Reply:
You're right that this new version might sound like a limitation.  However, I didn't understand the sense of accident in the original version, which is why I removed it.  Perhaps
```
> * one could accidentally copy and paste only a part of the code, and
```
would be best?

Reply:
Yes, I like the new suggestion - please update the PR with it!

Reply:
Done.